### ISSUE TITLE
formatter.js: Disabling quote doesn't work

### ISSUE THREAD

**Issue by TimPietrusky** [C2FO/fast-csv#97]
When I try to disable the quoting like this:

``` javascript
var csv = require('fast-csv');

var csvStream = csv.createWriteStream({
  quote : null
});
```

it's not working, because the `null` value of `options.quote` is ignored while setting `QUOTE` in [formatter.js](https://github.com/C2FO/fast-csv/blob/master/lib/formatter/formatter.js#L56):

``` javascript
QUOTE = options.quote || '"',
```

When I use the following option

``` javascript
var csvStream = csv.createWriteStream({
  quoteColumns : false
});
```

the fields are escaped / quoted, too. 
## Example

What I want: `my text;text with " in it;another text`
What I get: `my text;"text with "" in it";another text`

**Comment by pasynkov** [comment]
+1

**Comment by dustinsmith1024** [comment]
If you guys submit a PR we can get it reviewed and merged.  I don't think anyone is actively working on this.

**Comment by aalexgabi** [comment]
+1

**Comment by doug-martin** [comment]
Fixed with `v3.4.0` you can now set `quote` to `false` to disable all quoting.
